gke-deploy/core/gcs/gcs.go

```diff
@@ -0,0 +1,91 @@
+// Package gcs contains logic related to Google Cloud Storage.
+package gcs
+
+import (
+	"context"
+	"errors"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-builders/gke-deploy/services"
+)
+
+const (
+	timeoutErr = "GCS timeout"
+)
+
+var (
+	defaultTimeout = 1 * time.Hour
+	defaultDelay   = 2 * time.Second
+	errGCSTimeout  = errors.New(timeoutErr)
+)
+
+type GCS struct {
+	Timeout    time.Duration
+	Retries    int
+	Delay      time.Duration
+	GcsService services.GcsService
```

Comment:
Did you intentionally make these public again?

Reply:
Yes. I feel that it makes more sense to me to keep them public. It makes this class easier to use. I'm ok to make a change if you guys are strongly against this pattern.